testing/features/components/step_definitions/disclosure_steps.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Given("the Example Disclosure component is on the page") do
+  @component = Disclosure::Example.new.tap(&:load)
+end
+
+When("I open/close the disclosure component") do
```

Comment:
This works but I don't fully get how it works - i.e how open/close can then be translated to two expressions later (i.e `I open the disclosure component` and `I close the disclosure component`. Links to docs would be greatly appreciated - I've tried to look myself, but I'm probably not wording the search term properly.

Reply:
These are cucumber expressions. The only docs I could find are the Java one but the syntax is the same https://github.com/cucumber/cucumber-expressions#readme